opus/application/test_api/test_cart_api.py

```diff
@@ -2373,22 +2373,97 @@ def test__api_cart_datacsv_multi(self):
         self._run_json_equal(url, expected)
 
 
-            ################################################
-            ######### /__cart/view.html: API TESTS #########
-            ################################################
-
-    # XXX Need to implement tests
-
-
             ####################################################
             ######### /__cart/download.json: API TESTS #########
             ####################################################
+    def test__api_cart_download_single_no_hierarchical(self):
+        "[test_cart_api.py] /__cart/download.json: single opus id  & no hierarchical"
+        url = '/__cart/add.json?opusid=co-iss-n1462840881&reqno=456'
```

Comment:
Should call reset.json at the beginning of each test like the other existing tests do